search/v2/geocoder.cpp

```diff
@@ -684,5 +742,28 @@ coding::CompressedBitVector const * Geocoder::LoadStreets(MwmContext & context)
   m_streetsCache[mwmId] = move(streetsList[0]);
   return result;
 }
+
+coding::CompressedBitVector const * Geocoder::RetrieveGeometryFeatures(
+    MwmContext const & context, m2::RectD const & rect, int id)
+{
+  /// @todo
+  /// - Implement more smart strategy according to id.
+  /// - Move all rect limits here
+
+  auto & featuresV = m_geometryFeatures[context.m_id];
```

Comment:
s/featuresV/features/?
We're usually do not include variable's type into variable's name.